### PR TITLE
Comments: Load optimization

### PR DESCRIPTION
**Pull request by dellagustin**
With this PR, the comments are pushed to the UI via a chunked HTTP request, as soon as their replies are loaded. There is further room for optimization by sending the comments and replies in different steps.

Related issues:
- Fixes - https://github.com/Podcastindex-org/web-ui/issues/228

### CODE REVIEW

**Comment by dellagustin** [comment]
https://github.com/Podcastindex-org/web-ui/pull/236/commits/39fa9eafe86fad4916cdac86f36ae38cf3bbc1c6 removes some left over logs I used for debugging.

I recommend merging with "Squash and merge".

File: ui/src/components/Comments/index.tsx [hunk 1]
```diff
@@ -115,30 +125,70 @@ export default class Comments extends React.PureComponent<IProps, IState> {
...
+                    thisComponent.setState({
+                        loadingComments: false
+                    });
+                    return;
+                }
+                const parsedChunk = JSON.parse(new TextDecoder().decode(value));
+                
+                updateResponseBody(responseBody, parsedChunk);
+
+                console.log(responseBody);
```
Comment (ericpp) [inline ui/src/components/Comments/index.tsx:144]: Should this be logged to the console?
```suggestion
```

File: ui/src/components/Comments/index.tsx [hunk 2]
```diff
@@ -115,30 +125,70 @@ export default class Comments extends React.PureComponent<IProps, IState> {
...
+            const reader = response.body.getReader();
 
-            stateToSet.comments = responseBody.roots.map((root) => Comments.buildStateComment(root, responseBody));
-        }
+            const thisComponent = this;
 
-        this.setState(stateToSet);
+            await reader.read().then(function processChunk({done, value}) {
+                if(done) {
+                    console.log('done');
```
Comment (ericpp) [inline ui/src/components/Comments/index.tsx:134]: Was this part of debugging or intended?
```suggestion
```

**Comment by daveajones** [comment]
Will push this to production tomorrow!

**Comment by dellagustin** [comment]
That's great!

**Pull request merged**